default.nix

```diff
@@ -28,6 +28,9 @@ let
       pkgs.jq
```

Comment:
```suggestion
      pkgs.jq
      pkgs.coq
```

Reply:
we should consider making coq an optional dependency